i18n/de_DE.csv

```diff
@@ -128,3 +128,4 @@
 "Merchant data sharing","Gemeinsame Nutzung von Händlerdaten"
 "We enable Alma to anonymously retrieve, analyse the usage of our payment methods and share with me insights in order to increase my overall performance.","Wir ermöglichen es Alma, die Nutzung unserer Zahlungsmethoden anonymisiert abzurufen, zu analysieren und mir Erkenntnisse mitzuteilen, um meine Gesamtleistung zu steigern."
 "Alma Payment is expired","Alma Die Zahlung ist abgelaufen"
+"Attention: You have to enable the plugin for accepting payments. Meanwhile, you can complete your configuration.","Achtung! Sie müssen das Plugin für die Annahme von Zahlungen aktivieren. In der Zwischenzeit können Sie Ihre Konfiguration abschließen."
```

Comment:
Curious.
Crowdin hasn't translated the sentences ?

Reply:
are you sure ? 
"Attention: You have to enable the plugin for accepting payments. Meanwhile, you can complete your configuration.","Achtung! Sie müssen das Plugin für die Annahme von Zahlungen aktivieren. In der Zwischenzeit können Sie Ihre Konfiguration abschließen."